src/Ocelot/Cache/Middleware/OutputCacheMiddleware.cs

```diff
@@ -47,7 +48,8 @@ public class OutputCacheMiddleware : OcelotMiddleware
             {
                 _logger.LogDebug("cache entry exists for {downstreamUrlKey}", downstreamUrlKey);
 
-                SetHttpResponseMessageThisRequest(cached);
+				var response = await CreateHttpResponseMessage(cached);
```

Comment:
spacing seems a little off! Might just be github

Reply:
No it's not github, I probably have different settings on my visual studio and messes things up.
What are your settings on TextEditor -> C# -> Tabs?

Reply:
Maybe we can add a .editorconfig to enforce specific identation settings on the solution?

Reply:
Yeh good idea!

Reply:
i use vscode for Ocelot so not sure what I'm using...tbh im a shitty developer who doesnt care about tabs and spaces :E

Reply:
Do you have anything you would recommend for a default .editorconfig?